### PR TITLE
Say SDM not automation client

### PR DESCRIPTION
**Pull request by jessitron**
this is not a complete solution, but OK to merge at any time, will continue to work on it

### CODE REVIEW

File: docs/developer/client.md [hunk 1]
```diff
@@ -1,93 +1,85 @@
 When running a Software Delivery Machine (SDM) or lower-level Atomist
 API client, you interact with the Atomist development automation API
```
Comment (ddgenome) [inline docs/developer/client.md]: "Atomist development automation API" should probably be "Atomist API for software".

File: docs/developer/client.md [hunk 2]
```diff
@@ -1,93 +1,85 @@
 When running a Software Delivery Machine (SDM) or lower-level Atomist
```
Comment (ddgenome) [inline docs/developer/client.md]: We should probably rename the file and its title in the mkdocs.yml.

File: docs/developer/client.md [hunk 3]
```diff
@@ -1,93 +1,85 @@
...
 
-## Creating a client project
+## Creating an SDM project
 
 An Atomist API client project is any project that connects to the
-Atomist development automation API.  The reference implementation of
-the automation client is the [automation-client-ts][client-ts]
-library, which is written in [TypeScript][ts].  Client projects, like
-any SDM, depends on the automation-client library and uses its
+Atomist development automation API.  The reference implementation is the [automation-client-ts][client-ts]
```
Comment (ddgenome) [inline docs/developer/client.md]: "Atomist API for software"

File: docs/developer/client.md [hunk 4]
```diff
@@ -1,93 +1,85 @@
...
-experience, with excellent tooling and debugging support.
+TypeScript and [GraphQL][gql] provides a development
+experience with excellent tooling and debugging support.
 
-There are a few ways to create a new automation client project.  We
-suggest using the [automation-seed-ts][seed] project as a seed for
-your automation client project.
+There are a few ways to create a new SDM project.  We
+suggest using the [blank-sdm-seed][seed] project as a seed for
+your automation client project. You can do this localluy with the Atomist CLI: `atomist create sdm`.
```
Comment (ddgenome) [inline docs/developer/client.md]: spelling: `localluy`

Comment (ddgenome) [reply docs/developer/client.md]: Also best to break the command line into a code block so people can easily cut and paste.

```
atomist create sdm
```

File: docs/developer/client.md [hunk 5]
```diff
@@ -1,93 +1,85 @@
...
+## Creating an SDM project
 
 An Atomist API client project is any project that connects to the
-Atomist development automation API.  The reference implementation of
-the automation client is the [automation-client-ts][client-ts]
-library, which is written in [TypeScript][ts].  Client projects, like
-any SDM, depends on the automation-client library and uses its
+Atomist development automation API.  The reference implementation is the [automation-client-ts][client-ts]
+library, which is written in [TypeScript][ts]. Any SDM
+ depends on the @atomist/automation-client library and uses its
```
Comment (ddgenome) [inline docs/developer/client.md]: Why the leading space here?

File: docs/developer/client.md [hunk 6]
```diff
@@ -1,93 +1,85 @@
...
+[seed]: https://github.com/atomist/blank-sdm-seed (Blank SDM Seed Project)
 
 ### Slack
 
-You can create your very own automation client project using the
-Atomist bot.  You can run this bot command, which itself is
+If you use the Atomist service and Slack integration, 
+you can create your very own SDM project using the
+Atomist bot. This will make a repository in your SCM (GitHub, BitBucket, or GitLab).
+ You can run this bot command, which itself is
```
Comment (ddgenome) [inline docs/developer/client.md]: Leading space?

File: docs/developer/client.md [hunk 7]
```diff
@@ -207,27 +178,26 @@ persistent applications.
...
 occurs.
 
-### Multiple identical clients register
+### Multiple identical SDMs register
 
-If another client with the same name and version, typically obtained
-from the `package.json` "name" and "version" properties, registers,
-then all of the registered identical clients will receive the events
+If another client with the same name and version (typically obtained
+from the `package.json` "name" and "version" properties) registers,
```
Comment (ddgenome) [inline docs/developer/client.md]: Personally, I think parentheses should be avoided. Good for LISP, bad for prose.

File: docs/developer/client.md [hunk 8]
```diff
@@ -260,35 +230,29 @@ The `package.json` file defines the metadata and dependencies for the
...
-`npm start` | start the Atomist automation client
+`npm start` | start the SDM
 `npm test` | run tests
 
-### build
-
-The `build` directory contains the JavaScript sources output from
-TypeScript compilation.
-
 ### config
```
Comment (ddgenome) [inline docs/developer/client.md]: We can remove the config section. We no longer use it.

File: docs/developer/client.md [hunk 9]
```diff
@@ -311,30 +275,21 @@ project's [TypeDoc][typedoc].
 
 [typedoc]: http://typedoc.org/ (TypeDoc)
 
-### src
+### lib
```
Comment (ddgenome) [inline docs/developer/client.md]: This should probably move above lib/graphql.

File: docs/developer/client.md [hunk 10]
```diff
@@ -311,30 +275,21 @@ project's [TypeDoc][typedoc].
...
-#### events
+The `atomist.config.ts` file contains 
+project-specific configuration. This is the starting point when you want to look at what this SDM might do.
+See [SDM Configuration][sdm-config] for more details.
 
-The `src/events` directory contains the source code for event
-handlers.
+[sdm-config]: client.md#sdm-configuration (Atomist SDM Configuration)
 
 #### typings
```
Comment (ddgenome) [inline docs/developer/client.md]: If we have lib/graphql, this should probably be lib/typings and moved up after lib/graphql

File: docs/developer/commands.md [hunk 11]
```diff
@@ -22,7 +22,7 @@ In this guide you'll create MyCommandHandler, which responds to "do my thing".
 Command handlers are classes with a `handle` method and some
 decorators that supply metadata.  Store them anywhere in the `src`
 directory; your automation client discovers them on startup
-(or [specify them yourself](client.md#client-configuration)).
+(or [specify them yourself](client.md#sdm-configuration)).
```
Comment (ddgenome) [inline docs/developer/commands.md]: This could just used the named `[sdm-config]` link.

File: docs/quick-start.md [hunk 12]
```diff
@@ -1,125 +1,11 @@
-Develop your first automations: a bot command and an event handler.
+Develop your first Software Delivery Machine
 
-## Set up
+The easiest way to get started with Atomist is to start on your laptop, working with local commits. Then register your SDM with our service, integrating with Slack, your source control management, and your existing continuous integration tools.
 
-First, be sure someone in your Slack workspace has completed
-the [setup][setup].  Then, install [Node.js][node]
+The latest getting-started tutorials are over at [developer-tutorials](https://github.com/atomist/developer-tutorials/blob/master/README.md) for now. Issues and contributions welcome.
```
Comment (ddgenome) [inline docs/quick-start.md]: This README has nothing in it. Better to have the instructions here I would think.

Better to use a named link with a title.

Comment (jessitron) [reply docs/quick-start.md]: will discuss with Lieven tomorrow. Agree that it belongs here. Better to link to it than nothing, right now.
I have a PR in on that repo to put something in the README.

**Pull request merged**